### PR TITLE
feat: Add options for rate limit interval and burst

### PR DESCRIPTION
**Pull request by ephracis**
Enhancement:
Add two new variables, allowing the rate limit to be configured using the role.

Reason:
IHAC which needs these settings to be changed from their default values, and they would like to use RHEL System Roles as much as possible.

### CODE REVIEW

**Comment by spetrosi** [comment]
lgtm, please change a test for new variables.

**Comment by ephracis** [comment]
Thanks for the fast response. I added the variables to `tests_example.yml`. Please let me know if there's anything else.

**Comment by spetrosi** [comment]
[citest]

**Comment by spetrosi** [comment]
Please prefix the PR title with `feat: ` to indicate that this PR adds a new feature. And let's wait for the CI tests.

File: defaults/main.yml [hunk 1]
```diff
@@ -8,3 +8,5 @@ journald_per_user: false
 journald_compression: true
 journald_sync_interval: 0
 journald_forward_to_syslog: false
+journald_rate_limit_interval_sec: 30
+journald_rate_limit_burst: 10000
```
Comment (richm) [inline defaults/main.yml:12]: https://www.freedesktop.org/software/systemd/man/latest/journald.conf.html#RateLimitIntervalSec= 
good - we aren't changing the defaults with this, or changing any pre-existing behavior

One problem is that your text editor/IDE isn't adding a newline at the end of the file

File: templates/journald.conf.j2 [hunk 2]
```diff
@@ -36,3 +36,5 @@ RuntimeMaxFileSize={{ journald_max_file_size }}M
 {# settings below here apply to all storage types #}
 Compress={{ journald_compression | bool | ternary("yes", "no") }}
 ForwardToSyslog={{ journald_forward_to_syslog | bool | ternary("yes", "no") }}
+RateLimitIntervalSec={{ journald_rate_limit_interval_sec }}s
+RateLimitBurst={{ journald_rate_limit_burst }}
```
Comment (richm) [inline templates/journald.conf.j2:40]: newline here

Comment (ephracis) [reply templates/journald.conf.j2]: Fixed

File: defaults/main.yml (hunk 1 above)
Comment (ephracis) [reply defaults/main.yml]: Fixed

**Comment by richm** [comment]
[citest]

**Review by richm — APPROVED** [review]

**Pull request merged**